### DISCUSSION TITLE
Is there interest for realtime video nsfw filtering?

### DISCUSSION THREAD

**Discussion by realamirhe** [infinitered/nsfwjs#666] [Ideas]
I'm not sure this is a good place to ask, I wish there was a discord server.

Is there any interest in real-time movie frame filtering and controlling the film rating system of an online video based on NSFW detection?

**Comment by GantMan** [comment]
That's a pretty epic idea.   You'll have to build a node video parser.  But once you did, you could effectively use it on any AI model to find out all kinds of stuff about movies.

**Reply by realamirhe** [reply]
Thanks for the reply @GantMan.
I was thinking about sampling some frames ahead of the current frame in a browser-based extension.

But I guess the backend video parser would be a more appropriate action cause it might need more computation than expected in its action policy.
![network-simple](https://user-images.githubusercontent.com/23579958/192081873-829284f8-04ab-4c19-bdca-73ddc7086906.png)

I was wondering if there is any interest in the community to work on a project like this.

**Comment by GantMan** [comment]
This is such a cool idea!   But yeah, I'd start with a backend processor, then do front end to make it even more amazing.
